scripts/adaptive-sampling-integration-test.sh

```diff
@@ -2,16 +2,92 @@
 
 set -euf -o pipefail
 
-# This script is currently a placeholder.
+start_jaeger() {
+  echo "Starting Jaeger with adaptive sampling..."
+  SAMPLING_STORAGE_TYPE=memory SAMPLING_CONFIG_TYPE=adaptive go run -tags=ui ../cmd/all-in-one --log-level=debug &
```

Comment:
you need to clean up all background processes once the script exists (successfully or with error). Usually we use `trap` for that. I would recommend capturing PIDs into global variables and calling a function via `trap` that will will `kill -9` on all those PIDs (those that are not empty).